From 591774c2c6e8639ee0fa25af6669f72a5c93bd61 Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Thu, 18 Jan 2024 16:57:36 +0100
Subject: [PATCH] com.webos.service.location.perm.json: Update for LuneOS

Fixes:

WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg: 'telephony' being used in:
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg:   com.webos.service.location.perm.json
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg:   org.webosports.service.audio.perm.json
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg: 'tts.internal' being used in:
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg:   com.webos.service.configurator.perm.json
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg: 'tv.services' being used in:
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg:   org.webosports.service.audio.perm.json
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg: 'tv.settings' being used in:
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg:   com.webos.notification.perm.json
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg: 'wan.management' being used in:
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg:   com.webos.service.location.perm.json
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg: 'wan.query' being used in:
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg:   com.webos.service.location.perm.json

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Inappropriate [LuneOS specific]

 files/sysbus/com.webos.service.location.perm.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/files/sysbus/com.webos.service.location.perm.json b/files/sysbus/com.webos.service.location.perm.json
index 36e3afe..b320f2e 100644
--- a/files/sysbus/com.webos.service.location.perm.json
+++ b/files/sysbus/com.webos.service.location.perm.json
@@ -1,3 +1,3 @@
 {
-    "com.webos.service.location": ["networkconnection.management", "networkconnection.query", "wifi.query", "wifi.management", "wan.query", "wan.management", "telephony", "telephony.management", "settings.query"]
+    "com.webos.service.location": ["networkconnection.management", "networkconnection.query", "wifi.query", "wifi.management", "networking.query", "telephony.management", "settings.query"]
 }
